frontend/src/app/shared/components/file-links/file-link-list/file-link-list.html

```diff
@@ -35,5 +35,12 @@
       <span class="spot-icon spot-icon_external-link"></span>
       <span [textContent]="text.actions.linkFile(storageType)"></span>
     </a>
+    <a
+      class="spot-link op-files-tab--file-list-action-button"
+      (click)="openLinkFilesDialog()"
```

Comment:
If an element does not link to another route but instead performs an action, it should be a `button`. Could you change both this `a` and the one above to be a `button[type="button"]`?